### PR TITLE
Setup basics of PostsRestController

### PR DESCRIPTION
**Pull request by davefeucht**

### CODE REVIEW

File: src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java [hunk 1]
```diff
@@ -6,6 +6,11 @@
 final public class Utils {
     /** Value for a Category which has no parent */
     final public static int CATEGORY_NO_PARENT = -1; 
+    public enum EntityType {
```
Comment (eoikonomou) [inline src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java]: why is this not final?

Comment (davefeucht) [reply src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java]: I got a linting error about final not being applicable to type enum. I think by default enum values are always final.

Comment (davefeucht) [reply src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java]: `Illegal modifier for the member enum EntityType; only public, protected, private & static are permitted`

Comment (davefeucht) [reply src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java]: I will make it static though

Comment (eoikonomou) [reply src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java]: ah I see

**Comment by eoikonomou** [comment]
There are conflicts now :(

**Comment by davefeucht** [comment]
> There are conflicts now :(

Resolved the conflicts :)

**Pull request merged**